tasks/typescript/generate-typescript-annotations.js

```diff
@@ -55,7 +55,7 @@ function mod(yuidocs, localFileame, globalFilename, sourcePath) {
     String: 'string',
     Constant: 'any',
     undefined: 'undefined',
-    Null: 'null',
+    null: 'null',
```

Comment:
not sure how these changes relate to above. please separate into separate PR and tag with related issue.